### ISSUE TITLE
Template sorting

### ISSUE THREAD

**Issue by hauptrolle** [hauptrolle/chakra-templates#14]
# Feature request

## Describe the feature you need

It should be possible to sort templates.

**Comment by hauptrolle** [comment]
I guess the most simple solution would be to just add a prefix to the template names like `01_featureGrid`. So the Sorting in the fs and the website is the same.

**Comment by hauptrolle** [comment]
Another idea would be, to not use the filesystem as the source of truth, and just create a plain object which holds all the data. For example:

```ts
const data = [
  {
    name: "Page Sections",
    url: "page-sections",
    children: [
      {
        name: "Features",
        url: "features",
        children: [
          {
            name: "Grid list with heading",
            fileName: "gridListWithHeading",
          },
          {
            name: "Simple three column grid",
            fileName: "simpleThreeColumns",
          },
        ],
      },
    ],
  },
];
```

With this we are more flexibel for how we
* structure the urls
* define the headings for the example frames
* sort the templates

Downsides:
* this could be a mess when adding more and more templates
* fileName has to be exactly the same

What is your opinion on this subject @lazarnikolov94 ?

**Comment by lazarnikolov94** [comment]
I like the idea of structuring the data in a plain object. It does add a dependency when adding new templates, but on the other hand it'll give us a lot of room for custom features, such as easy sorting, IDs, tags maybe etc...

**Comment by hauptrolle** [comment]
Sounds good! Will try to refactor a little bit :)

**Comment by hauptrolle** [comment]
Fixed in #17
